src/Drupal/ConfigTrait.php

```
@@ -0,0 +1,478 @@
+   * Given the following config values:
+   *   | name              | key          | value                   |
+   *   | system.site       | name         | My site                 |
+   *   | mymodule.settings | api.endpoint | https://api.example.com |
+   *   | mymodule.settings | roles        | ["editor","reviewer"]   |
+   * @endcode
+   */
+  #[Given('the following config values:')]
+  public function configSetMultiple(TableNode $table): void {
+    foreach ($table->getHash() as $row) {
+      if (!isset($row['name'], $row['key']) || !array_key_exists('value', $row)) {
+        throw new \RuntimeException('The config values table must contain "name", "key" and "value" columns.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '123', column '19'). (undefined)

(MissingImport)